### DISCUSSION TITLE
HEAD requests to a CFC generate 500 error

### DISCUSSION THREAD

**Discussion by arielmaioli** [wheels-dev/wheels#1209] [General]
Hi guys!
A customer with CF11 asked me if I could change the response to a "200 OK" or "405 Method Not Allowed" when there is a HEAD type request to any CFC.
For example, if I make the following call: `curl --location --head https://example.com/bla/sample.cfc?method=test` I get a 500 (the same happens with any CFC that makes a HEAD call without specifying a method)
I searched this post on [StackOverFlow](https://stackoverflow.com/questions/74061026/a-head-call-to-a-cfc-always-returns-a-500) but I don't see any solution.
Can anyone help me with this?
Is it possible to change this from the CF side?

Thanks!!

**Comment by adamcameron** [comment]
Followed-up on Stack Overflow. Need some clarification pls.

**Reply by arielmaioli** [reply]
@adamcameron I answered you there, thanks so much for your time on this! let me know if you need more information please.
